test/LiveServer.hs

```diff
@@ -69,6 +69,9 @@ randomDataSrc s' = genBS s'
         else do yield $ concatIt byteArr64 oneMiB
                 genBS (s - oneMiB)
 
+lookupValue :: Text -> [(Text, Text)] -> Maybe Text
```

Comment:
you should use [lookup](https://www.stackage.org/haddock/lts-9.17/base-4.9.1.0/Prelude.html#v:lookup) instead.